commit 4b793d5344af26a0fca4f6d7ccc7ce0b55d3c935
Author: Scott Giese <scttgs0@gmail.com>
Date:   Sat Feb 2 19:46:04 2019 -0600

    Remove deprecated boost/utility.
    This is enable compatibility with boost 1.69.

diff --git a/simgear/nasal/cppbind/NasalHash.hxx b/simgear/nasal/cppbind/NasalHash.hxx
index aac672fd..5019d08b 100644
--- a/simgear/nasal/cppbind/NasalHash.hxx
+++ b/simgear/nasal/cppbind/NasalHash.hxx
@@ -24,6 +24,7 @@
 
 #include <simgear/structure/map.hxx>
 #include <boost/iterator/iterator_facade.hpp>
+#include <boost/core/enable_if.hpp>
 
 namespace nasal
 {
diff --git a/simgear/props/props.hxx b/simgear/props/props.hxx
index c94186cb..68d3c03c 100644
--- a/simgear/props/props.hxx
+++ b/simgear/props/props.hxx
@@ -55,13 +55,14 @@ namespace boost {
   struct disable_if : public disable_if_c<Cond::value, T> {};
 }
 #else
-# include <boost/utility.hpp>
 # include <boost/type_traits/is_enum.hpp>
+# include <boost/core/enable_if.hpp>
 
 # include <simgear/debug/logstream.hxx>
 # include <simgear/math/SGMathFwd.hxx>
 # include <simgear/math/sg_types.hxx>
 #endif
+
 #include <simgear/structure/SGReferenced.hxx>
 #include <simgear/structure/SGSharedPtr.hxx>
 
diff --git a/simgear/structure/SGWeakReferenced.hxx b/simgear/structure/SGWeakReferenced.hxx
index 3bb70ffa..27f647b4 100644
--- a/simgear/structure/SGWeakReferenced.hxx
+++ b/simgear/structure/SGWeakReferenced.hxx
@@ -22,7 +22,7 @@
 #include "SGSharedPtr.hxx"
 
 #include <boost/type_traits/is_base_of.hpp>
-#include <boost/utility/enable_if.hpp>
+#include <boost/core/enable_if.hpp>
 
 #ifdef _MSC_VER
 # pragma warning(push)
